<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReconstructionClass</class>
 <widget class="QMainWindow" name="ReconstructionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reconstruction</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="RayCastViewer" name="widgetRecon" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
   <widget class="RayCastViewer" name="widgetColor" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="DepthViewer" name="widgetDepth" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>240</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QRadioButton" name="rb_mode_loading">
       <property name="text">
        <string>Loading</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="rb_mode_live_saving">
       <property name="text">
        <string>Live &amp; Saving</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="rb_mode_live">
       <property name="text">
        <string>Live</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pb_play_pause">
       <property name="text">
        <string>Play/Pause</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pb_reset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="pb_save_mesh">
       <property name="text">
        <string>Save Mesh</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="cb_remove_dup">
       <property name="text">
        <string>Remove Duplicate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="cb_remove_iso">
       <property name="text">
        <string>Reomve Iso</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QCheckBox" name="cb_edge_collapse">
       <property name="text">
        <string>Edge Collapse</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>500</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pb_load_color_list">
       <property name="text">
        <string>Load Color List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_color_filter">
       <property name="text">
        <string>Color Filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RayCastViewer</class>
   <extends>QWidget</extends>
   <header>RayCastViewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DepthViewer</class>
   <extends>QWidget</extends>
   <header>DepthViewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="reconstruction.qrc"/>
 </resources>
 <connections/>
</ui>
